src/Commands.js

```diff
@@ -37,6 +38,7 @@
             fired = [],
             prefix = "cmdId_",
             indexer = 0,
+            rethrow, //default true
```

Comment:
first, i wasnt aware this would get into this PR as i made it before this change. I didnt think github will add any future commit to my master after making the PR... I wasnt planing on this being thrown at you yet... :)

regarding the rethrow, the problem is that chronos currently just swallows errors and thats problematic behavior. the very simplistic log mechanism in the eventsUtil isnt really a solution...

perhaps make it default to false and let the developer decide the behavior?